clang/test/Sema/c2x-nodiscard.c

```diff
@@ -65,15 +65,31 @@ void GH104391() {
   M; // expected-warning {{ignoring return value of function declared with 'nodiscard' attribute}}
 }
 
+struct S4 get_s_ignored(void) __attribute__((candiscard));
```

Comment:
There doesn't seem to be any testing for what happens if you add both `candiscard` and `nodiscard` to a function.